languageserver/src/main/scala/langserver/core/LanguageServer.scala

```diff
@@ -38,7 +39,8 @@ class LanguageServer(inStream: InputStream, outStream: OutputStream)(implicit s:
     case DidChangeTextDocumentParams(td, changes) => onChangeTextDocument(td, changes)
     case DidSaveTextDocumentParams(td) => onSaveTextDocument(td)
     case DidCloseTextDocumentParams(td) => onCloseTextDocument(td)
-    case DidChangeWatchedFiles(changes) => onChangeWatchedFiles(changes)
+    case DidChangeWatchedFilesParams(changes) => onChangeWatchedFiles(changes)
```

Comment:
I've also renamed `DidChangeWatchedFiles` to `DidChangeWatchedFilesParams` for consistency.